lib/ibanity/client.rb

```diff
@@ -75,14 +75,18 @@ def execute(method:, uri:, headers:, query_params: {}, payload: nil, idempotency
       JSON.parse(raw_response)
     end
 
-    def build_headers(customer_access_token = nil, idempotency_key = nil)
+    def build_headers(customer_access_token: nil, idempotency_key: nil, extra_headers: nil)
       headers = {
         content_type:  :json,
         accept:        :json,
       }
       headers["Authorization"]           = "Bearer #{customer_access_token}" unless customer_access_token.nil?
       headers["Ibanity-Idempotency-Key"] = idempotency_key unless idempotency_key.nil?
-      headers
+      if extra_headers.nil?
+        headers
+      else
+        headers.merge(extra_headers)
```

Comment:
What do you think about merging our headers into the extra_headers to be sure ours are used?